fast64_internal/oot/c_writer/oot_level_c.py

```diff
@@ -195,18 +195,23 @@ def ootRoomMeshToC(room, textureExportSettings):
 		raise PluginError("Error: Room " + str(room.index) + " has no mesh children.")
 
 	meshHeader = CData()
-	meshHeader.header = "extern MeshHeader" + str(mesh.meshType) + " " + mesh.headerName() + ';\n'
-	meshHeader.source = "MeshHeader" + str(mesh.meshType) + " " + mesh.headerName() + ' = ' +\
-		"{ {" + str(mesh.meshType) + "}, " + str(len(mesh.meshEntries)) + ", " +\
-		"(u32)&" + mesh.entriesName() + ", (u32)&(" + mesh.entriesName() + ") + " +\
-		"sizeof(" + mesh.entriesName() + ") };\n\n"
+	meshHeader.header = f"extern MeshHeader{mesh.meshType} {mesh.headerName()};\n"
+	meshHeader.source = '\n'.join((
+		f"MeshHeader{mesh.meshType} {mesh.headerName()} = {'{'}",
+		f"{indent}{mesh.meshType},",
+		f"{indent}{len(mesh.meshEntries)},",
+		f"{indent}{mesh.entriesName()},",
+		f"{indent}{mesh.entriesName()} + sizeof({mesh.entriesName()})",
+		f"{'}'};"
+	)) + "\n\n"
 
 	meshEntries = CData()
-	meshEntries.header = "extern MeshEntry" + str(mesh.meshType) + " " + mesh.entriesName() + "[" + str(len(mesh.meshEntries)) + "];\n"
-	meshEntries.source = "MeshEntry" + str(mesh.meshType) + " " + mesh.entriesName() + "[" + str(len(mesh.meshEntries)) + "] = {\n"
+	meshEntryType = f"MeshHeader{'01' if not int(mesh.meshType, base=10) else '2'}Entry "
```

Comment:
```suggestion
	meshEntryType = f"MeshHeader{'01' if mesh.meshType == '0' else '2'}Entry "
```
(what about mesh type 1 then, or is it handled somewhere else?)

Reply:
mesh type 1 isn't supported atm since, I assume, pre-rendered backgrounds aren't extracted yet, since there's a PR opened for that I think we should implement type 1

also could be cool to export skyboxes (idk how it works tho so idk if that'd be useful or not)